### PR TITLE
Port auto_vertical feature test from mycli to pgcli

### PR DESCRIPTION
**Pull request by meeuw**
## Description
Port auto_vertical feature test from mycli to pgcli

## Checklist

- [x] I've added this contribution to the `changelog.md`.

### CODE REVIEW

**Comment by tsroten** [comment]
@meeuw Good work!

I think the wording of this feature should match the config option in the config file (`expand` and `auto_expand`). I know that @amjith and @j-bennet talked about removing one of those config options, so it might be good to hear from them what the final wording will be.

File: changelog.rst [hunk 1]
```diff
@@ -8,6 +8,7 @@ Features:
 * Improved formatting of arrays in output (Thanks: `Joakim Koljonen`_)
 * Don't quote identifiers that are non-reserved keywords. (Thanks: `Joakim Koljonen`_)
 * Remove the ``...`` in the continuation prompt and use empty space instead. (Thanks: `Amjith Ramanujam`_)
+* Port auto_vertical feature test from mycli to pgcli. (Thanks: `Dick Marinus`_)
```
Comment (tsroten) [inline changelog.rst]: Since the user-facing change is a new command-line option, maybe we can reword this to not be about the tests?

File: pgcli/main.py [hunk 2]
```diff
@@ -793,11 +795,13 @@ def get_last_query(self):
 @click.option('--prompt', help='Prompt format (Default: "\\u@\\h:\\d> ").')
 @click.option('-l', '--list', 'list_databases', is_flag=True, help='list '
               'available databases, then exit.')
+@click.option('--auto-vertical-output', is_flag=True,
+              help='Automatically switch to vertical output mode if the result is wider than the terminal width.')
```
Comment (tsroten) [inline pgcli/main.py]: This should probably say "expanded mode" and not "vertical output mode". Just a minor difference between mysql and psql.

Comment (meeuw) [inline pgcli/main.py]: Hmm, I would like to keep the `.feature` files the same for mycli and pgcli. Am I something missing that "auto expand" and "vertical output" are synonymous?

**Comment by j-bennet** [comment]
@meeuw @tsroten Yes, we were talking about having `expand` set to one of `"on", "off", "auto"`, and removing `auto_expand`. I think it would be a good change, to keep the exploding number of config options at bay.

**Comment by j-bennet** [comment]
@meeuw Sorry this PR has been sitting a while, I think it's fine to merge. We can refactor `expand` and `auto_expand` separately from this. This branch only needs merge conflicts resolved.

**Comment by meeuw** [comment]
@j-bennet thanks for the reminder; I've rebased the branch

**Comment by j-bennet** [comment]
👍 Thank you!
